src/clandinin_lab_to_nwb/brezovec/brezovecimagingextractor.py

```diff
@@ -47,8 +47,10 @@ def _parse_xml(folder_path: PathType) -> ElementTree.Element:
     tree = ElementTree.parse(xml_file_path)
     return tree.getroot()
 
-class BrezovecMultiPlaneImagingExtractor(MultiImagingExtractor):
+
+class BrezovecMultiPlaneImagingExtractor(ImagingExtractor):
```

Comment:
We should inherit from the simple Imaging extractor, Bruker does it this way because of the file per plane convention. We already have everything here. 
@alessandratrapani